
float32 error_x
float32 error_y
float32 error_z
float32 error_yaw

float32 d_error_x
float32 d_error_y
float32 d_error_z
float32 d_error_yaw

float32 proj_error_x
float32 proj_error_y

float32 pterm_x
float32 pterm_y
float32 pterm_z
float32 pterm_yaw

float32 dterm_x
float32 dterm_y
float32 dterm_z
float32 dterm_yaw

float32 vel_x
float32 vel_y
float32 vel_z
float32 vel_yaw